src/index.js

```diff
@@ -40,13 +51,19 @@ export class Wizard extends Component {
     Public Methods
   \***************************************************************************/
 
-  setStep = (stepId, newState) => {
+  componentDidMount () {
```

Comment:
componentDidMount shouldn't be used in this case as it only serves to waste update cycles. By the time state is defined, props is already available to us so we can set this by changing the state property to:

```
state = {
  currentStep: this.props.defaultStep || this.steps[0]
  ...
}
```

Reply:
Legit.